sync/datahub/metrichub_glossary.py

```diff
@@ -20,19 +21,27 @@ def _build_metric_dict(metric: MetricHubDefinition) -> Dict:
     metric_content = ""
 
     if metric.deprecated:
-        metric_content += "⚠️ **This metric has been deprecated**\n\n"
+        metric_content += "#### ⚠️ **This metric has been deprecated**\n\n"
 
+    if metric.friendly_name:
+        metric_content += f"## {metric.friendly_name} \n\n"
+    
     if metric.level:
-        metric_content += f"Metric Level: {_get_metric_level_link_text(metric.level)}"
+        metric_content += f"**Metric Level:** {_get_metric_level_link_text(metric.level)}\n\n"
 
     if metric.description:
-        metric_content += f"_{metric.description.strip()}_\n\n"
+        metric_content += f"{metric.description.strip().replace(linesep, ' ')}\n\n"
```

Comment:
nit: I haven't looked at examples here but maybe `. ` as a sentence separator? Feel free to ignore 
```suggestion
        metric_content += f"{metric.description.strip().replace(linesep, '. ')}\n\n"
```

Reply:
Yeah, in the metrichub definitions I've looked it, it looks like people are adding line breaks when the lines are too long and/or when they're ends of sentences and in the latter case they include a period, so I don't think we need to include `.` here.